Client/Frontend/Browser/EnhancedTrackingProtection/EnhancedTrackingProtectionDetailsVC.swift

```diff
@@ -0,0 +1,186 @@
+
+    // MARK: - UI
+
+    private let siteTitleLabel: UILabel = .build { label in
+        label.font = ETPMenuUX.Fonts.websiteTitle
+    }
+
+    private var closeButton: UIButton = .build { button in
+        button.layer.cornerRadius = 0.5 * ETPMenuUX.UX.closeButtonSize
+        button.clipsToBounds = true
+        button.setTitle(.AppSettingsDone, for: .normal)
+        button.titleLabel?.font = ETPMenuUX.Fonts.viewTitleLabels
+        button.setTitleColor(.systemBlue, for: .normal)
+    }
+
+    private let siteInfoSection = ETPSectionView(frame: .zero)
+
+    private let siteInfoImage: UIImageView = .build { image in
+        image.contentMode = .scaleAspectFit
+        image.clipsToBounds = true
+        image.layer.masksToBounds = true
+        image.layer.cornerRadius = 5
+    }
+
+    private var siteInfoTitleLabel: UILabel = .build { label in
+        label.font = ETPMenuUX.Fonts.viewTitleLabels
+        label.numberOfLines = 0
+    }
+
+    private var siteInfoURLLabel: UILabel = .build { label in
+        label.font = ETPMenuUX.Fonts.minorInfoLabel
+        label.numberOfLines = 0
+    }
+
+    private let connectionView = ETPSectionView(frame: .zero)
+
+    private let connectionImage: UIImageView = .build { image in
+        image.contentMode = .scaleAspectFit
+        image.clipsToBounds = true
+        image.layer.masksToBounds = true
+        image.layer.cornerRadius = 5
+    }
+
+    private let connectionStatusLabel: UILabel = .build { label in
+        label.font = ETPMenuUX.Fonts.viewTitleLabels
+    }
+
+    private let connectionVerifierLabel: UILabel = .build { label in
+        label.font = ETPMenuUX.Fonts.minorInfoLabel
+        label.isHidden = true
+    }
+
+    // MARK: - Variables
+    var viewModel: EnhancedTrackingProtectionDetailsVM
+
+    // MARK: - View Lifecycle
+
+    init(viewModel: EnhancedTrackingProtectionDetailsVM) {
+        self.viewModel = viewModel
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupView()
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        updateViewDetails()
+        applyTheme()
+    }
+
+    private func setupView() {
+        view.addSubview(siteTitleLabel)
+        view.addSubview(closeButton)
+        closeButton.addTarget(self, action: #selector(closeButtonTapped), for: .touchUpInside)
+
+        siteInfoSection.addSubview(siteInfoImage)
+        siteInfoSection.addSubview(siteInfoTitleLabel)
+        siteInfoSection.addSubview(siteInfoURLLabel)
+        view.addSubview(siteInfoSection)
+
+        connectionView.addSubview(connectionImage)
+        connectionView.addSubview(connectionStatusLabel)
+        connectionView.addSubview(connectionVerifierLabel)
+        view.addSubview(connectionView)
```

Comment:
You can replace these multiple lines of `addSubview` with .`addSubviews()`. See View.swift for more info on this convenience func.